bugbug/bug_features.py

```diff
@@ -690,3 +691,194 @@ def apply_rollback(bugs_iter):
             bugs_iter = apply_rollback(bugs_iter)
 
         return pd.DataFrame(apply_transform(bug) for bug in bugs_iter)
+
+
+class IsPerformanceBug(SingleBugFeature):
+    """Determine if the bug is related to performance based on given bug data."""
+
+    name = "Is Performance Bug"
+    type_name = "performance"
+    keywords = {"perf", "topperf", "main-thread-io"}
+
+    def __call__(
+        self,
+        bug: bugzilla.BugDict,
+        bug_map: Optional[dict[int, bugzilla.BugDict]] = None,
+    ) -> bool:
+        if any(
+            f"[{whiteboard_text}" in bug["whiteboard"].lower()
+            for whiteboard_text in (
```

Comment:
I would change it to be `prefix`, similar to the keywords:
```suggestion
            for prefix in (
```